src/d/actor/d_a_tag_attention.cpp

```diff
@@ -19,13 +22,13 @@ static dCcD_SrcSph sph_check_src = {
         /* SrcObjTg  SPrm    */ 0,
         /* SrcObjCo  SPrm    */ cCcD_CoSPrm_Set_e | cCcD_CoSPrm_IsOther_e | cCcD_CoSPrm_NoCrr_e,
         /* SrcGObjAt Se      */ 0,
-        /* SrcGObjAt HitMark */ dCcG_AtHitMark_None_e,
-        /* SrcGObjAt Spl     */ dCcG_At_Spl_UNK0,
+        /* SrcGObjAt HitMark */ 0,
```

Comment:
Is there a reason you changed this? Those enum values here before are equivalent to 0